### PR TITLE
PMM-4809 Add flag max table stats

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-4809

related: 
 - https://github.com/percona/pmm-managed/pull/292
 - https://github.com/percona/pmm-agent/pull/132
 - https://github.com/percona/pmm-admin/pull/75
 - https://github.com/percona/pmm/pull/451

FB: https://github.com/Percona-Lab/pmm-submodules/pull/522

### CODE REVIEW

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-451/

File: api/swagger/swagger.json [hunk 1]
```diff
@@ -9824,6 +9819,11 @@
                     }
                   }
                 },
+                "node_id": {
+                  "description": "Node identifier on which a service is been running.\nExactly one of these parameters should be present: node_id, node_name, add_node.",
+                  "type": "string",
+                  "x-order": 0
+                },
```
Comment (AlekSi) [inline api/swagger/swagger.json]: @BupycHuk May you take a look:
* why this definition jumped there?
* why `add_node` does not have `x-order` at all?

Comment (BupycHuk) [reply api/swagger/swagger.json]: ok, I'll take a look

Comment (BupycHuk) [reply api/swagger/swagger.json]: it's because swagger flatten command replaces whole block.

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-451/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-451/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-451/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-451/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-451/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-451/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-451/

**Comment by AlekSi** [comment]
https://github.com/percona/pmm/pull/461

**Pull request closed without merging**